client/actions/form.js

```diff
@@ -2,6 +2,13 @@
 
 import { redirect } from "next/navigation";
 import { serverUrl } from "@/constants";
+import {
+  formatName,
+  formatAddress,
+  formatDateCreated,
+  formatPhone,
+  formatTime,
+} from "@/lib/fetchAppointments";
```

Comment:
Feels like it'd be nicer to expose a single `formatAppointment` from lib if that's where formatting logic is gonna sit. Right now, half it it's here and half there.